hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson

```diff
@@ -61,6 +61,12 @@
       uvm_test_seq: sysrst_ctrl_combo_detect_ec_rst_vseq
       reseed: 1
     }
+    {
+      name: sysrst_ctrl_override_test
+      uvm_test_seq: sysrst_ctrl_pin_override_vseq
+      reseed: 1
```

Comment:
the reseed numbers here are too small. recommend to just use default value

Reply:
Changed the seed value to default value